### DISCUSSION TITLE
Rules engine on Array inputs & rules on array

### DISCUSSION THREAD

**Discussion by Meenakshise** [microsoft/RulesEngine#339] [Q&A]
If I have an user object with multiple Orders as an array list with properties (Order Id, Order amount and order date). And I have a rule which should be success only if the user has any order which exceeds 10000 on the order amount. Is that possible ?
Mainly on the array list both at the rule level and at the input level?

**Comment by abbasc52** [comment]
@Meenakshise It is possible. RulesEngine has decent support for linq related operators. I have not understood your request completely but something like this should work in rulesengine:-
Expression : `user.Orders.Any(c => c.Amount > 10000)`

If this is not what you are looking for, Please provide additional context.
